doc/02-spacegraphcats-use-cases.md

```diff
@@ -0,0 +1,307 @@
+        trim-low-abund.py --gzip -C 3 -Z 18 -M 20e9 -V - -o SRR1976948.abundtrim.fq.gz
+```
+
+Download the query genome (this is a metagenome bin produced from SRR1976948):
+```
+wget ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/508/995/GCA_001508995.1_ASM150899v1/GCA_001508995.1_ASM150899v1_genomic.fna.gz
+```
+
+Configuration file, saved as `conf1.yml`.
+Note that configuration files may contain multiple input sequences from which to build the cDBG and catlas, or multiple search fasta files. 
+
+```
+catlas_base: 'SRR1976948'
+input_sequences:
+- SRR1976948.abundtrim.fq.gz
+ksize: 31
+radius: 1
+search:
+- GCA_001508995.1_ASM150899v1_genomic.fna.gz
+searchquick: GCA_001508995.1_ASM150899v1_genomic.fna.gz
+```
+
+run spacegraphcats:
+
+```
+python -m spacegraphcats run conf1.yml extract_contigs extract_reads --nolock 
+```
+
+compare the output sequences:
+
+```
+sourmash compute -k 21,31,51 --scaled 2000 -o GCA_001508995.1_ASM150899v1_genomic.sig GCA_001508995.1_ASM150899v1_genomic.fna.gz
+sourmash compare -k 31 --csv comp.csv *sig SRR1976948_k31_r1_search_oh0/*sig
+```
+
+Publications using this approach are listed below:
+
+1. Brown, C.T., Moritz, D., O’Brien, M.P. et al. Exploring neighborhoods in large metagenome assembly graphs using spacegraphcats reveals hidden sequence diversity. Genome Biol 21, 164 (2020). https://doi.org/10.1186/s13059-020-02066-4
+2. Lumian, J.E., Jungblut, A.D., Dillion, M.L. et al. Metabolic Capacity of the Antarctic Cyanobacterium Phormidium pseudopriestleyi That Sustains Oxygenic Photosynthesis in the Presence of Hydrogen Sulfide. Genes 12, 426 (2021). https://doi.org/10.3390/genes12030426 
+
+### Identifying the context of e.g. horizontally transferred genes
+
+Spacegraphcats queries don't need to be an entire genome, but can be any size (greater than *k*). 
+For example, one can query with a gene of interest. 
+In the example below, we identify antibiotic resistance genes in time series human stool metagenomes using [GROOT](https://github.com/will-rowe/groot), and then use the identified genes as spacegraphcats queries.
+Using this method, we can see how the context of antibiotic resistance genes changes over time.
+
+The dataset used below is a time series from a single individual with Crohn's disease, sequenced by the [Integrative Human Microbiome Project (iHMP)](https://ibdmdb.org/).
+The stool microbiome from this individual was sequenced at 12 different timepoints over 37 weeks, and the individual was on antibiotics at various times during the that timeframe.
+
+![](https://i.imgur.com/H7JiYvr.png) *Times of sequencing and antibiotic exposure for individual H4017 from the Integrative Human Microbiome Project.*
+
+We'll determine the sequence context of one antibiotic resistance gene in one sample using one radius. 
+
+
+Install companion software needed for this analysis:
+```
+conda install sourmash=4.0.0 fastp=0.20.1 bbmap=38.70 khmer=3.0.0a3 groot=1.1.2 samtools=1.12 bandage=0.8.1 blast=2.11.0
```

Comment:
sourmash deps could be changed here, but I guess it doesn't really matter.

Reply:
to the range, e.g. `sourmash>=4.1.0,<5` ?

Reply:
yep